### DISCUSSION TITLE
Cannot find curl extension for php

### DISCUSSION THREAD

**Discussion by ayDavidGitHere** [termux/termux-packages#12887] [Q&A] [answered]
Is it possible to use curl with php on termux.
The commands to install curl e.g `apt-get install php7-curl` does not exist.

**Comment by sk0kanik** [comment, marked as answer]
On termux, main php package also includes curl extension. 727704d

**Reply by ayDavidGitHere** [reply]
No, what I have is this when i use the curl extension.

`PHP Warning:  PHP Startup: Unable to load dynamic library 'curl' (tried: /data/data/com.termux/files/usr/lib/php/curl (dlopen failed: library &quot;/data/data/com.termux/files/usr/lib/php/curl&quot; not found), /data/data/com.termux/files/usr/lib/php/curl.so (dlopen failed: library &quot;/data/data/com.termux/files/usr/lib/php/curl.so&quot; not found)) in Unknown on line 0`

**Reply by xtkoba** [reply]
What does your code look like? The example shown in https://www.geeksforgeeks.org/how-to-enable-curl-in-php/ works for me (with `php7`).

**Reply by ayDavidGitHere** [reply]
I updated the php version, curl works now.
But the warning message still persists when i start the server.


`PHP Warning: PHP Startup: Unable to load dynamic library 'curl' (tried: /data/data/com.termux/files/usr/lib/php/curl (dlopen failed: library &quot;/data/data/com.termux/files/usr/lib/php/curl&quot; not found), /data/data/com.termux/files/usr/lib/php/curl.so (dlopen failed: library &quot;/data/data/com.termux/files/usr/lib/php/curl.so&quot; not found)) in Unknown on line 0`
